ShellLR: 
  color: [0.23375, 0.47125, 0.495]
  center_height: 0.015
  min_thickness: 5e-3
  dist_thershold: 0.3
  max_dimension: 0.2
  # Optional for few objects
  fixed_position: [-0.375, -0.207]
  orientation_xyz: [0, 0, 0.2618]
  center_offset: [0.04, -0.02]
  max_vel_scaling: 0.5

ShellUR: 
  color: [0.23375, 0.47125, 0.495]
  center_height: 0.015
  min_thickness: 5e-3
  dist_thershold: 0.3
  max_dimension: 0.2
  # Optional for few objects
  fixed_position: [-0.361, -0.035]
  orientation_xyz: [0, 0, 0.2618]
  center_offset: [0.04, -0.02]
  max_vel_scaling: 0.5

ShellLL: 
  color: [0.23375, 0.47125, 0.495]
  center_height: 0.015
  min_thickness: 5e-3
  dist_thershold: 0.3
  max_dimension: 0.2
  # Optional for few objects
  fixed_position: [-0.61, -0.224]
  orientation_xyz: [0, 0, 0.2618]
  center_offset: [0.04, -0.02]
  max_vel_scaling: 0.5

ShellUL: 
  color: [0.23375, 0.47125, 0.495]
  center_height: 0.015
  min_thickness: 5e-3
  dist_thershold: 0.3
  max_dimension: 0.2
  # Optional for few objects
  fixed_position: [-0.614, -0.035]
  orientation_xyz: [0, 0, 0.2618]
  center_offset: [0.04, -0.02]
  max_vel_scaling: 0.5

BatteryBlue:
  color: [0.455, 0.5775, 0.7165] 
  center_height: 0.03
  min_thickness: 5e-3
  dist_thershold: 0.3
  max_dimension: 0.15
  # Optional for few objects
  orientation_xyz: [0, 0, 1.5708]
  max_vel_scaling: 0.6

ScrewBowlBlue:
  color: [0.556, 0.7035, 0.846]
  center_height: 0.035
  min_thickness: 2.5e-3
  dist_thershold: 0.1
  max_dimension: 0.15
  # Optional for few objects
  orientation_xyz: [0., 0., 0.] #[0, 0, 1.5708]
  center_offset: [0, -0.04] #[-0.04, 0]
  max_vel_scaling: 0.5

BatteryYellow:
  color: [0.63925, 0.664, 0.402] 
  center_height: 0.02
  min_thickness: 5e-3
  dist_thershold: 0.3
  max_dimension: 0.15
  # Optional for few objects
  orientation_xyz: [0., 0., 0.] #[0, 0, 1.5708]
  max_vel_scaling: 0.6

ScrewBowlYellow:
  color: [0.8925, 0.912, 0.6875]
  center_height: 0.035
  min_thickness: 2.5e-3
  dist_thershold: 0.1
  max_dimension: 0.15
  # Optional for few objects
  orientation_xyz: [0, 0, 1.5708]
  center_offset: [0.05, 0]
  max_vel_scaling: 0.5
